### ISSUE TITLE
InteractiveShell Completer attribute is uppercase despite being an instance.

### ISSUE THREAD

**Issue by Carreau** [ipython/ipython#9507]
I'm perturbed when I see that. it hurts my brain and my feelings.

Should be replaced/shimmed/ deprecated/.

**Comment by takluyver** [comment]
Yeah, there are several of the older bits of code that don't follow modern capitalisation conventions. They were quite probably written before those conventions were established, and now it's not worth the hassle of changing them until we're actually refactoring the relevant bits of code.
